tests/conftest.py

```diff
@@ -17,6 +17,7 @@
 MY_BUCKET = 'streaming-test-bucket'
 MY_PREFIX = 'train'
 GCS_URL = 'https://storage.googleapis.com'
+R2_URL = 'https://c4c9097c86eeef198b16fd088f5f99d6.r2.cloudflarestorage.com'
```

Comment:
Is this a real account id? Can we use any other alphanumeric value such as `abcdefghijkl1234567` ? since we are anyhow mocking the response!!

Reply:
Random value gives SSL error and test fails, I've replaced it with just `'https://r2.cloudflarestorage.com'`

Reply:
Addressed as part of https://github.com/mosaicml/streaming/pull/255/commits/3b7e2068f524d6ff2a3b9de4908ff7064a8eb5eb.